FtpClientApp/FtpClientApp/Program.cs

```diff
@@ -215,7 +215,27 @@ public static bool GetResponce(String username, String password, String server)
                     MyAnswer = false;
                     break;
                 case "2":
-                    Console.WriteLine(" Not Implemented Yet \n");
+                    Console.Clear();
+                    Console.WriteLine(" You chose: Upload file to Remote server  \n");
+                    //File upload
+                    FileUpload uploadfile = new FileUpload(conn);
+                    FtpTestWrapper wrapper_file = new FtpTestWrapper();
+                    Console.WriteLine(" ** Specify file to be uploaded \n (Mention absolute path in this format, for ex: C:/xyz/rst/abc.filetype. Filetypes accepted: .txt, .jpg, .png ** \n");
+                    String filetobeuploaded = uploadfile.getFileName();
+                    Console.WriteLine(" \n ** Specify valid location on server where file is to be uploaded (for ex: ftp://localhost/test) \n");
```

Comment:
You shouldn't tell them to specify their location on the server using "(for ex: ftp://localhost/test)" The ftp://localhost can be added from the serverconnectionInfo instance variable `connection`. Then you just have to combine the server info from that class with the file path the enter. So you should just ask for the directory instead of the full on the server.

Reply:
Fixed